core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/edit-ema-toolbar/edit-ema-toolbar.component.html

```diff
@@ -1,10 +1,79 @@
-<ng-content select="[title]"></ng-content>
-<p-toolbar>
-    <div class="p-toolbar-group-left" data-testId="toolbar-left-content">
-        <ng-content select="[left]"></ng-content>
-    </div>
-    <div class="p-toolbar-group-right" data-testId="toolbar-right-content">
-        <ng-content select="[right]"></ng-content>
-    </div>
-</p-toolbar>
-<ng-content select="[bottom]"></ng-content>
+<ng-container *ngIf="editorState$ | async as es">
```

Comment:
Maybe we can create another selector

Reply:
No thank you, this store is already too complicated 😅